muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/guider.js

```diff
@@ -90,7 +90,10 @@ $(document).ready(function(){
         }
         mApi().user.users.read({searchString : params, archetype : 'STUDENT', maxResults: 25 })
         .callback(function (err, users) {
-          users.userCount = users.length;         
+          
+          if(users != undefined){
```

Comment:
It would be better to use typeof (users != undefined) also matches false, 0, empty array, etc,,,